atmo/settings.py

```diff
@@ -428,6 +428,11 @@ def post_setup(cls):
 
 class Test(Dev):
     """Configuration to be used during testing"""
+    CONSTANCE_BACKEND = 'constance.backends.redisd.RedisBackend'
```

Comment:
So far we've treated Redis as a non-persistent data store and the database as the single source of truth, which was the reason to use the database backend (with the Redis cache as the cache in front). I'm not sure if we should use the Redis backend for constance simply because I don't know how safe the AWS Redis cluster we use is.